apps/arweave/src/ar_data_sync.erl

```diff
@@ -329,10 +329,38 @@ get_chunk(Offset, #{ packing := Packing } = Options) ->
 			false ->
 				Offset
 		end,
+	IsMinerRequest = maps:get(is_miner_request, Options, false),
 	case IsRecorded of
 		{{true, StoredPacking}, StoreID} ->
-			get_chunk(Offset, SeekOffset, Pack, Packing, StoredPacking, StoreID);
+			get_chunk(Offset, SeekOffset, Pack, Packing, StoredPacking, StoreID,
+					IsMinerRequest);
+		{true, StoreID} ->
+			case IsMinerRequest of
+				false ->
+					ok;
+				true ->
+					UnpackedReply = ar_sync_record:is_recorded(Offset, {?MODULE, unpacked}),
+					?LOG_ERROR([{event, chunk_record_not_associated_with_packing},
+							{tags, [solution_proofs]},
+							{store_id, StoreID},
+							{seek_offset, SeekOffset},
+							{is_recorded_unpacked, io_lib:format("~p", [UnpackedReply])}])
+			end,
+			{error, chunk_not_found};
 		_ ->
```

Comment:
Would it be worth also logging the value of the reply here? In case it's something unexpected?

Reply:
also maybe the result of `ets:lookup(sync_records, {ID, StoreID})`?

I'm wondering if the `sync_records` table got corrupt, maybe the `{ID, StoreID}` is not responding correctly